packages/server/__tests__/api/keywords.test.js

```diff
@@ -34,10 +33,31 @@ describe('`/api/keywords` endpoint', async () => {
         },
     };
 
+    let testKeywordsByAgency = null;
+
     before(async function beforeHook() {
         this.timeout(9000); // Getting session cookies can exceed default timeout.
-        fetchOptions.admin.headers.cookie = await getSessionCookie('michael@nv.gov');
+        fetchOptions.admin.headers.cookie = await getSessionCookie('admin1@nv.gov');
         fetchOptions.staff.headers.cookie = await getSessionCookie('user1@nv.gov');
+
+        // Tests below assume the presence of at least one keyword per agency. Previously, we had
+        // a db seed that created some default COVID-related keywords but now that that is gone, we
+        // create some here to avoid dependence on the global keywords seed.
+        // Note: this does not dedupe agency IDs and that's by design; some of the tests need to
+        // delete multiple keywords from the same agency.
+        const allAgencyIds = Object.values(agencies.admin).concat(Object.values(agencies.staff));
+        const testKeywords = allAgencyIds.map((agency_id) => ({
+            mode: 'autoinsert ALL keywords matches',
+            search_term: 'test_keyword',
+            notes: '',
+            agency_id,
+        }));
+        const createdKeywords = await knex('keywords')
+            .insert(testKeywords)
```

Comment:
Hmm, might need to clean this up after the test since other tests call e.g. `getGrants` which queries filtered based on the keywords table.